content/02_KeyTypes.md

```diff
@@ -1,488 +1,627 @@
-The `IObservable<T>` interface is the dominant type that you will be exposed to for representing a sequence of data in motion, and therefore will comprise the core concern for most of your work with Rx and most of this book.
-
----
-
-<div class="webonly">
-	<h1 class="ignoreToc">Additional recommended reading</h1>
-	<div align="center">
-		<!--Head First Design Patterns (Kindle) Amazon.co.uk-->
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B00AA36RZY&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-
-		<!--Design Patterns (Kindle) Amazon.co.uk-->
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B000SEIBB8&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-
-		<!--Clean Code (Kindle) Amazon.co.uk-->
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=0132350882&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-		
-		<!--C# 3.0 Design Patterns (Kindle) Amazon.co.uk-->
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B0043EWUAC&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-	</div>
</div>
+---
+title : Key Types
+---
+
+# Key types
+
+Rx is a powerful framework that can greatly simplify code that responds to events. But to write good Reactive code you have to understand the basic concepts. The fundamental building block of Rx is an interface called `IObservable<T>`. Understanding this, and its counterpart `IObserver<T>`, is the key to success with Rx.
```

Comment:
Once we get out of this intro material, and into the section dedicated to the specific interfaces, I say this:

> that it is part of the .NET runtime libraries

So I already do. (Except I always avoid the term BCL because it's ambiguous. For example, according to the ECMA spec, `IObservable<T>` very definitely isn't in the BCL, because the BCL is actually a very narrow subset of the types in the runtime class libraries.)